.gitignore

```diff
@@ -15,7 +15,6 @@ cmd/wfx-viewer/wfx-viewer
 coverage*
 *.log
 /results
-generated/wfx.swagger.yml
```

Comment:
Hm, what about the output of the new openapiv3? Shouldn't that be ignored as well?

Reply:
No since we no longer have to pre-process the swagger yaml :)